python/pfs/drp/stella/fitBroadbandSED.py

```diff
@@ -88,7 +98,7 @@ def runDataRef(self, dataRef):
         pfsConfig = dataRef.get("pfsConfig")
         return self.run(pfsConfig)
 
-    def run(self, pfsConfig):
+    def run(self, pfsConfig: PfsConfig) -> List[Union[NDArray[numpy.float64], None]]:
```

Comment:
I think this would read easier as `List[Optional[NDArray[np.float64]]]`. You can also use `List[NDArray[np.float64] | None]`.

Reply:
To use the disjunction operator `|`, we need python-3.10. We are using python-3.8. The future introduction of the construction `A|None` is the reason why I use `Union[A, None]` instead of `Optional[A]`. I don't think `Optional` will survive long.

Reply:
Oh, sorry. I crossed wires with LSST, which has moved to python 3.10. We'll inherit that eventually, but not yet.